src/lib.rs

```diff
@@ -435,6 +435,21 @@ mod tests {
         }
     }
 
+    #[test]
+    #[serial]
+    #[ignore]
```

Comment:
You're ignoring this test because we don't have trezor emulator support right? If so please just add an [ignore](https://doc.rust-lang.org/reference/attributes/testing.html#the-ignore-attribute) comment to that effect here so others know the reason.